Changelog.md

```diff
@@ -1,6 +1,11 @@
 # Next
 
+- **Breaking Change** Renamed all occurrences of `queue` to `callbackQueue`.
+- Add optional callback queue parameter to reactive providers.
+- Enabled the "Allow app extension API only" flag.
```

Comment:
Oh, oops. Started working on this and it seems like I did so from the wrong branch. I'll remove those.